EcomLogin/src/components/login.component.js

```diff
@@ -0,0 +1,34 @@
+import React, { Component } from "react";
+
+export default class Login extends Component {
+    render() {
+        return (
+            <form>
```

Comment:
not using react bootstrap? Also, this JSX should go in frontend/src/components/Login/Login.js

Reply:
also no form validation or submission handling?